### PR TITLE
Refine references to TFS and rename files

### PR DESCRIPTION
**Pull request by wpschaub**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Guidance/blob/master/.github/CLA.md).

Changes proposed in this pull request:  
- Renamed files to *-azure-DevOps-*
- Replace TFS references in guidance to Azure DevOps Server

@ALM-Rangers/guidance

### CODE REVIEW

File: src/Planning/understand-options-azure-devops-environments.md [hunk 1]
```diff
@@ -25,17 +25,17 @@ While both Azure DevOps Server and Azure DevOps provide the same essential servi
 
 ## Deployment Options
 
-[Azure DevOps Server](https://visualstudio.microsoft.com/tfs) (TFS) and Visual Studio Azure DevOps Server (Azure DevOps ) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
+[Azure DevOps Server](https://visualstudio.microsoft.com/tfs) and Visual Studio Azure DevOps Server (Azure DevOps ) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
```
Comment (NielZeeman) [inline src/Planning/understand-options-azure-devops-environments.md]: Change to : https://visualstudio.microsoft.com/team-services/ ?

File: src/Planning/understand-options-azure-devops-environments.md [hunk 2]
```diff
@@ -25,17 +25,17 @@ While both Azure DevOps Server and Azure DevOps provide the same essential servi
...
 
 [Azure DevOps](https://visualstudio.microsoft.com/team-services/) is the recommended software as a service (SaaS) cloud offering on the far right.
```
Comment (NielZeeman) [inline src/Planning/understand-options-azure-devops-environments.md]: change to https://azure.microsoft.com/en-us/services/devops

**Comment by jesfer** [comment]
Unless it is part of some other pr please fix the names of the actual links we are providing urls for... "Visual Studio Azure DevOps Server (Azure DevOps)" smells of too many search and replace actions ;)

File: src/Planning/understand-options-azure-devops-environments.md (hunk 1 above)
Comment (wpschaub) [reply src/Planning/understand-options-azure-devops-environments.md]: I'll change it to https://azure.microsoft.com/en-us/services/devops/server/

File: src/Planning/understand-options-azure-devops-environments.md (hunk 2 above)
Comment (wpschaub) [reply src/Planning/understand-options-azure-devops-environments.md]: I changed TFS to https://azure.microsoft.com/en-us/services/devops/server/ and AzDO to the *devops link

File: src/Planning/understand-options-azure-devops-environments.md [hunk 3]
```diff
@@ -25,17 +25,17 @@ While both Azure DevOps Server and Azure DevOps provide the same essential servi
 
 ## Deployment Options
 
-[Azure DevOps Server](https://visualstudio.microsoft.com/tfs) (TFS) and Visual Studio Azure DevOps Server (Azure DevOps ) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
+[Azure DevOps Server](https://azure.microsoft.com/en-us/services/devops/server/) and [Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
```
Comment (nschonni) [inline src/Planning/understand-options-azure-devops-environments.md]:
```suggestion
[Azure DevOps Server](https://azure.microsoft.com/services/devops/server/) and [Azure DevOps](https://azure.microsoft.com/services/devops/) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/azure/devops/user-guide/services?view=azure-devops).
```

File: src/Planning/understand-options-azure-devops-environments.md [hunk 4]
```diff
@@ -54,20 +59,20 @@ The following table is a quick summary of the three deployment options, with a c
 |Multi Servers (scale out)|730+|3600+|
 
 >
-> [!TIP]
+> [!NOTE]
 >
-> The dual and multi server options allow you to scale your TFS environment as your organization or number of power users grow. You can change the deployment option and migrate to Azure DevOps  at a later stage. See [Migration options](https://docs.microsoft.com/en-us/vsts/articles/migrate-to-vsts-from-tfs?view=vsts) to review migration approaches.
+> The dual and multi server options allow you to scale your Azure DevOps Server environment as your organization or number of power users grow. You can change the deployment option and migrate to Azure DevOps  at a later stage. See [Migration options](https://docs.microsoft.com/en-ca/azure/devops/migrate/migrate-from-tfs?view=azure-devops&viewFallbackFrom=vsts) to review migration approaches.
```
Comment (nschonni) [inline src/Planning/understand-options-azure-devops-environments.md]:
```suggestion
> The dual and multi server options allow you to scale your Azure DevOps Server environment as your organization or number of power users grow. You can change the deployment option and migrate to Azure DevOps  at a later stage. See [Migration options](https://docs.microsoft.com/azure/devops/migrate/migrate-from-tfs?view=azure-devops) to review migration approaches.
```

File: src/Planning/understand-options-azure-devops-environments.md [hunk 5]
```diff
@@ -54,20 +59,20 @@ The following table is a quick summary of the three deployment options, with a c
...
 >
 
-If you have remote sites, sharing code with your main TFS server site, the [Azure DevOps Server Proxy](https://docs.microsoft.com/en-us/tfs/server/install/install-proxy-setup-remote) helps to cache recently used version control files, improve remote user experience, and reduce network load.
+If you have remote sites, sharing code with your main Azure DevOps server site, the [Azure DevOps Server Proxy](https://docs.microsoft.com/en-ca/azure/devops/server/install/install-proxy-setup-remote?view=azure-devops) helps to cache recently used version control files, improve remote user experience, and reduce network load.
```
Comment (nschonni) [inline src/Planning/understand-options-azure-devops-environments.md]:
```suggestion
If you have remote sites, sharing code with your main Azure DevOps server site, the [Azure DevOps Server Proxy](https://docs.microsoft.com/azure/devops/server/install/install-proxy-setup-remote?view=azure-devops) helps to cache recently used version control files, improve remote user experience, and reduce network load.
```

File: src/Planning/understand-options-azure-devops-environments.md (hunk 3 above)
Comment (wpschaub) [reply src/Planning/understand-options-azure-devops-environments.md]: Good feedback. Dropped locale from all URLs.

File: src/Planning/understand-options-azure-devops-environments.md (hunk 5 above)
Comment (wpschaub) [reply src/Planning/understand-options-azure-devops-environments.md]: Good feedback. Dropped locale from all URLs.

**Comment by wpschaub** [comment]
@kenmuse are you OK if I submit a pull request to MS docs to get the planning article and your workbook into the editorial process, or d you want to make changes before? Once published to docs, we can make changes there.

**Pull request merged**